koku/providers/aws/aws_provider.py

```diff
@@ -135,31 +135,37 @@ def name(self):
 
     def cost_usage_source_is_reachable(self, credential_name, storage_resource_name):
         """Verify that the S3 bucket exists and is reachable."""
+        if not credential_name or len(credential_name.strip()) == 0:
```

Comment:
You could use `.isspace()` here.

(See: https://docs.python.org/3.6/library/stdtypes.html#str.isspace )

Reply:
Good to know 👍